influence_benchmark/RL/EI.py

```diff
@@ -1,23 +1,16 @@
 import json
 
 from influence_benchmark.RL.base_iteration import BaseIteration
-from influence_benchmark.stats.preferences_per_iteration import get_best_worst_n_trajectories
 
 
 class ExpertIteration(BaseIteration):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def _select_and_format_trajectories(self, trajectory_iteration_dir):
-        selected_trajectories, _ = get_best_worst_n_trajectories(
-            trajectory_iteration_dir, self.top_n_trajs_per_initial_state, final_reward=self.final_reward
-        )
-
-        self._format_and_save_trajectories_for_sft(selected_trajectories, trajectory_iteration_dir)
-
-    def _format_and_save_trajectories_for_sft(self, selected_partial_trajs, partial_traj_folder):
+    def _format_and_save_trajectories(self, selected_partial_trajs, partial_traj_folder):
```

Comment:
What does the "partial" mean in this name? Is it because it contains the best/worst and we only need the best?

Reply:
It's technically not really entire trajectories, because each json will contain partial trajectories: trajectories up until a certain turn. Like if you take the length of this json, it will be a lot longer than the number of trajs (it's the number of timesteps)

Reply:
I see, shouldn't the "_partial" be there in the base_iteration.py and KTO.py classes as well then? Looks like it's not there now

Reply:
Yeah maybe we should make the same change elsewhere